internal/handlers/hana/msg_logout.go

```diff
@@ -17,11 +17,11 @@ package hana
 import (
 	"context"
 
-	"github.com/FerretDB/FerretDB/internal/handlers/common"
```

Comment:
"Common packages are bad, they say"
- Hold my beer, it is `commoncommands` now
:D

Reply:
Kind of, yeah :) But `common` grew too big…